quiz.js

```diff
@@ -37,11 +52,31 @@ db.dropDatabase();
 
 // your code begins here
 
-
-
-
-
-// your code ends here
+db.houses.update({
+  name: 'House Arryn'
+  }, {
+    $push: {person: 'Jon Arryn'
+    }
+});
+
+db.houses.update({
+  name: 'House Stark'
+  }, {
+    $push: {person: 'Ned Stark',
+            person: 'Arya Stark',
+            person: 'Sansa Stark'
+    }
+});
+
+db.houses.update({
+  name: 'House Targaryen'
+  }, {
+    $push: {person: 'Daenerys Targaryen',
+            person: 'Viserys Targaryen'
+    }
+});
+
+//your code ends here
```

Comment:
The second and third ones are wrong. To push multiple values to `person` (which, incidentally, should probably have been named `people`), you need to either (1) run separate $push commands for each person or (2) use the $each operator, as follows:
```
db.houses.update({ name : 'House Stark' }, { $push : { person : 
  {$each : [
      'Ned Stark',
      'Arya Stark',
      'Sansa Stark'
  ] } 
} } );
```